src/main/java/com/epam/aidial/core/controller/ApplicationController.java

```diff
@@ -22,98 +22,67 @@ public class ApplicationController {
 
     private final ProxyContext context;
     private final Vertx vertx;
-    private final CustomApplicationService customApplicationService;
-    private final boolean includeCustomApplications;
+    private final ApplicationService applications;
 
     public ApplicationController(ProxyContext context, Proxy proxy) {
         this.context = context;
         this.vertx = proxy.getVertx();
-        this.customApplicationService = proxy.getCustomApplicationService();
-        this.includeCustomApplications = customApplicationService.includeCustomApplications();
+        this.applications = proxy.getApplicationService();
```

Comment:
I wouldn't recommend to have the field in a plural form. It may confuse a reviewer.
Let's add a suffix `Service`.
`applications` are usually treated as a set or list of applications.

Reply:
renamed